storage/src/vespa/storage/storageserver/statemanager.cpp

```diff
@@ -647,4 +652,12 @@ void StateManager::immediately_send_get_node_state_replies() {
     sendGetNodeStateReplies();
 }
 
+void
+StateManager::request_almost_immediate_node_state_replies()
+{
+    std::unique_lock guard(_threadLock);
+    _requested_almost_immediate_node_state_replies.store(true, std::memory_order_relaxed);
```

Comment:
Should we consider clearing `_controllers_observed_explicit_node_state` here as we do in `immediately_send_get_node_state_replies()` to handle the case where a cluster controller does not have a pending GetNodeState at the time of next worker thread tick?

Reply:
Fixed.